fractal_server/app/models/project.py

```diff
@@ -24,13 +24,15 @@ class Project(_ProjectBase, SQLModel, table=True):
     )
 
     dataset_list: list[Dataset] = Relationship(
+        back_populates="project",
```

Comment:
TBD whether this is needed

Reply:
Not needed - see test introduced with [07955b1](https://github.com/fractal-analytics-platform/fractal-server/pull/1082/commits/07955b11aefa0e80890340b569e403cdcf7f3d0b).